test/uk/gov/hmrc/customsservicestatus/controllers/services/CheckServiceSpec.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.customsservicestatus.controllers.services
+
+import org.mockito.Mockito.when
+import uk.gov.hmrc.customservicestatus.errorhandlers.CustomsServiceStatusError.ServiceNotConfiguredError
+import uk.gov.hmrc.customsservicestatus.controllers.helpers.BaseSpec
+import uk.gov.hmrc.customsservicestatus.models.{CustomsServiceStatus, Status}
+import uk.gov.hmrc.customsservicestatus.repositories.CustomsServiceStatusRepository
+import uk.gov.hmrc.customsservicestatus.services.CustomsServiceStatusService
+
+import java.time.Instant
+import scala.concurrent.Future
+
+class CheckServiceSpec extends BaseSpec {
+  val repo    = mock[CustomsServiceStatusRepository]
+  val service = new CustomsServiceStatusService(repo)
+
+  "check" should {
```

Comment:
method will no longer be called "check" (once change made), so Spec needs to change.

Reply:
changed.